docs/release-notes/2.33.0.md

```diff
@@ -0,0 +1,145 @@
+
+- [PMM-8516](https://jira.percona.com/browse/PMM-8516): [Backup Management]:  When creating an a backup, the Vendor field was not automatically populated in the Back-up on demand dialog. This issue is now fixed. 
+
+- [PMM-10627](https://jira.percona.com/browse/PMM-10627): [DBaaS]: You can now create a single node cluster for PSMDB. 
+ 
+ - [PMM-10855](https://jira.percona.com/browse/PMM-10855): Avoid massive gRPC messages formatting for disabled log level
+
+
+- [PMM-10903](https://jira.percona.com/browse/PMM-10903): [DBaaS]: To simplify DBaas, you no longer have to manually enter the Public Address on the **PMM Settings > Advanced Settings** page. With DBaaS enabled, PMM will automatically detect and populate the **Public Address**.
+
+
+- [PMM-11085](https://jira.percona.com/browse/PMM-11085): [Backup Management]: We've removed the **Create backup** option under **All backups** page > **Actions** menu because this option can only create on-demand backups. You can continue to create both on-demand and scheduled backups using the **Create backup** button on this page. 
+
+- [PMM-10881](https://jira.percona.com/browse/PMM-10881): [CVE]: As part of PMM 2.33.0, grafana was updated to 9.2.5, which fixes some critical vulnerabilities. For more information, see [grafana 9.2.5](https://grafana.com/blog/2022/11/08/security-release-new-versions-of-grafana-with-critical-and-moderate-fixes-for-cve-2022-39328-cve-2022-39307-and-cve-2022-39306/).
+
+
+- [PMM-10135](https://jira.percona.com/browse/PMM-10135): Due to its limited use, the **Examples** tab has been disabled in PMM 2.33.0. Also, to safeguard our user's privacy, we have removed Personally Identifiable Information (PII) from the **Explain** tab.
+
+
+- [PMM-8655](https://jira.percona.com/browse/PMM-8655): Latest state of agents is now available on the PMM server after the connection is re-established between the client and server.
+
+- [PMM-10969](https://jira.percona.com/browse/PMM-10969): VictoriaMetrics operator has been upgraded from 0.24 to 0.29.0.
+
+
+- [PMM-10554](https://jira.percona.com/browse/PMM-10554): Postgres collector now provides runtime duration for scrapes at the collector level. As a result, we can identify collectors who take too long. By utilizing this data, corrective action can be taken.
+
+
+
+- [PMM-10629](https://jira.percona.com/browse/PMM-10629): [Components Upgrade]: VictoriaMetrics has been upgraded from 1.77.1 to 1.82.1.
+
+
+
+
+## Bugs Fixed
+
+- [PMM-10683](https://jira.percona.com/browse/PMM-10858): In specific environments, including AWS, some EPEL repository mirrors did not respond within the time limit defined by `pmm-update`, currently set to 30 seconds. This was causing `supervisord` to kill `pmm-update-checker`, the component responsible for verifying if a newer version of the PMM Server is available for upgrade. This issue has been resolved now.
```

Comment:
This should be [PMM-10858] instead of [PMM-PMM-10683]

Reply:
Thanks will fix it.